de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
@sri2k1us I'm not entirely sure I understand the suggestion. When a collaborator is deleted, do you want to unshare everything that the user has ever shared with that collaborator individually? We don't really have a good way to track that at the moment. In other words, we don't have a way to definitively know whether permission for the collaborator to access a resource was granted directly by the current user or not.

We could create a way to track that. The easiest way that I can think of is to create a special group for each collaborator (containing only that collaborator) and have the permissions that would be assigned to that collaborator assigned to that group instead. Then when a user deletes a collaborator, the special group could be deleted, which would effectively unshare everything that the user has ever shared with that collaborator.